### ISSUE TITLE
How many epochs did you use to gain 93% accuracy?

### ISSUE THREAD

**Issue by ThanhBarTN** [GantMan/nsfw_model#67]
Hi @TechnikEmpire , 
I'm using your tensorflow 2.x scripts to train on my own dataset. I'm wondering how many epochs did you use to gain 93% accuracy for 5 classes of images
